discovery/db_discovery.go

```diff
@@ -99,13 +99,25 @@ func (dbo *DBOrchestratorPoolCache) GetOrchestrators(numOrchestrators int) ([]*n
 	pred := func(info *net.OrchestratorInfo) bool {
 
 		if err := dbo.ticketParamsValidator.ValidateTicketParams(pmTicketParams(info.TicketParams)); err != nil {
+			glog.V(common.DEBUG).Infof("unable to validate ticket params - orchestrator=%v recipient=%v err=%v",
```

Comment:
Not sure how useful including the recipient address is here. Also need to use `.Hex()` if we do include it or else we'll just print a slice.

Reply:
Yeah at this point I'm not certain what our go-to orchestrator identification method is: by address or by service URI. I assume service URI would be a better fit as most will eventually end up using a domain name.